WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
Should a warning be thrown here that indicating that the `tax_query` whitelist comment is deprecated ?

(if not, you may want to fold the two blocks into one)